servers/graphql-kotlin-spring-server/src/main/kotlin/com/expediagroup/graphql/server/spring/extensions/generatorExtensions.kt

```diff
@@ -25,6 +25,13 @@ import org.springframework.aop.support.AopUtils
  * the schema generator can use
  */
 internal fun List<Any>.toTopLevelObjects() = this.map {
+    it.toTopLevelObject()
+}
+
+/**
+ * Converts given object to a [TopLevelObject] wrapper.
+ */
+internal fun Any.toTopLevelObject() = this.let {
```

Comment:
nip: would it be helpful to be explicit about the return type ? the name of the function pretty much says it, but specifying the return type of function wouldn't harm